tools/who_what_benchmark/whowhatbench/visualtext_evaluator.py

```diff
@@ -30,6 +30,19 @@ def prepare_default_data(num_samples=None):
     )
 
 
+def align_phi3_v(model_type, tokenizer):
```

Comment:
```suggestion
def extra_generation_config(model_type, tokenizer):
```
? to be more generic

Reply:
Why should it be more generic? The current name limits the responsibility so it's easier to ignore calls if the reader isn't interested in phi3_v

Reply:
This also attracts user attention since need to understand whether he / she needs to add `align_<other model>`

Maybe it's nor very clear why is it called `align_` ..

Reply:
Would `fix_phi3_v_eos_token_id()` be better?

Reply:
yes, but all those comments are minor..

So, up to you